### ISSUE TITLE
1.4版本中的Chapter11中的第2个

### ISSUE THREAD

**Issue by QLMX** [caicloud/tensorflow-tutorial#94]
运行出现ValueError: Both labels and logits must be provided.错误，这该怎样改正啊

**Comment by perhapszzy** [comment]
你要先运行4.1，再运行4.2

**Comment by 124399839** [comment]
Chapter11为什么tensorboard不能可视化，graph看不到，
